sqlite/expr.c

```diff
@@ -5661,6 +5661,21 @@ static char* sqlite3ExprDescribe_inner(
     case TK_FUNCTION: {
       char *ret, *ret2;
       int i;
+      /* dh: not all the functions can be blindely passed remotely! */
+      if(strncmp(pExpr->u.zToken, "now", 3) == 0)
```

Comment:
Make this strncasecmp as well.
Also, the comparison should perhaps be on the length of zToken as strncmp("nowzzz", "now", 3) would also return 0 (unless parser catches it beforehand). Ditto for the cmp below.